sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -117,6 +117,24 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
       .getOne();
   }
 
+  /**
+   * Get the Education Program of an Application.
+   * if the Education Program is not active
+   *  return null
+   * else
+   *  return the Education Program row.
+   * @param educationProgramId Selected Form ProgramYear of the application.
+   */
+  async getActiveEducationProgram(
+    educationProgramId: number,
+  ): Promise<EducationProgram> {
+    return this.repo
+      .createQueryBuilder("programs")
+      .where("programs.is_active = true")
+      .andWhere("programs.id  = :educationProgramId", { educationProgramId })
+      .getOne();
+  }
+
```

Comment:
Is there a reason for not using the existing method? 

![image](https://github.com/bcgov/SIMS/assets/54600590/a406aca0-2907-4892-9d98-414173c34205)